File: 프로그래머스/unrated/181902. 문자 개수 세기/문자 개수 세기.java
import java.util.Arrays;
import java.util.HashMap;
import java.util.Map;

class Solution {
    public int[] solution(String my_string) {
        Map<Character, Integer> abc = new HashMap<>();

        for (int i=65; i<123; i++) {
            if (i < 91 || 96 < i) abc.put((char) i, 0);
        }

        for (int i=0; i<my_string.length(); i++) {
            char ch = my_string.charAt(i);
            
            if (abc.containsKey(ch)) {
                abc.put(ch, abc.get(ch)+1);
            }
        }

        return abc.values().stream().mapToInt(e->e).toArray();
    }
}

File: 프로그래머스/unrated/181902. 문자 개수 세기/문자 개수 세기.py
def solution(my_string):
    answer = [0]*52
    
    for ch in my_string:
        if ch.isupper():
            answer[ord(ch) - ord('A')] += 1
        else:
            answer[ord(ch) - ord('G')] += 1
            
    return answer